cartography/intel/azure/sql.py

```diff
@@ -0,0 +1,742 @@
+import logging
+from azure.mgmt.sql import SqlManagementClient
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_client(credentials, subscription_id):
+    client = SqlManagementClient(credentials, subscription_id)
+    return client
+
+
+@timeit
+def get_server_list(credentials, subscription_id):
+    try:
+        client = get_client(credentials, subscription_id)
+        server_list = list(client.servers.list())
+
+    except Exception as e:
+        print(f'Failed to retrieve servers : {e}')
```

Comment:
instead of prints, we should use the logger to print out the exception details

Reply:
done